README.md

```diff
@@ -1,3 +1,8 @@
 # heroku-deploy
 
 A Dockerfile that we use to deploy stuff to Heroku with.
+
+### Configuration
+
+1. Expose heroku credentials to the container via: `HEROKU_LOGIN` and `HEROKU_API_KEY` variables (set in CircleCI's UI or `config.yml`)
+2. Execute `setup-heroku.sh` that stores the credentials in `~/.netrc` before calling any script.
```

Comment:
Change the scripts so they call `setup-heroku.sh` first? Then we can lose this instruction.

Reply:
It also means we don't need to do this step in all the circle.yml's?

Reply:
It also means we don't need to do it in all the circle.yml's?

Reply:
Sounds good, I'll change it!